app/views/admin/items/_form.html.erb

```diff
@@ -0,0 +1,20 @@
+<%= bootstrap_form_for @item, :html => { multipart: true }  do |f| %>
+  <%= f.text_field :title %>
+  <%= f.text_area :description %>
+  <%= f.number_field :unit_price, label: "Price (in cents)"%>
```

Comment:
Maybe this should be label: "Price"? (and calculate the :unit_price on the backend)?